Kickstarter-iOS/Views/Controllers/SettingsViewController.swift

```diff
@@ -153,13 +154,30 @@ extension SettingsViewController: UITableViewDelegate {
     return SettingsSectionType.sectionHeaderHeight
   }
 
+  func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+    return tableView.dequeueReusableHeaderFooterView(withIdentifier: Nib.SettingsHeaderView.rawValue)
+  }
+
   func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
-    return 0.1 // Required to remove the footer in UITableViewStyleGrouped
+    guard section == SettingsSectionType.ratingAppVersion.rawValue else { return 0.1 }
+
+    return UITableView.automaticDimension
   }
 
-  func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+  func tableView(_ tableView: UITableView, estimatedHeightForFooterInSection section: Int) -> CGFloat {
+    return 44
```

Comment:
Can we make this a static variable in `SettingsFooterView`? Something like `SettingsFooterView.defaultHeight`?